ui/page/proposal/proposal_details_page.go

```diff
@@ -93,10 +101,46 @@ func (pg *proposalDetails) Handle() {
 	}
 
 	if pg.vote.Button.Clicked() {
-		newVoteModal(pg.Load).Show()
+		newVoteModal(pg.Load, pg.proposal).Show()
+	}
+
+	for pg.viewInPoliteiaBtn.Clicked() {
+		components.GoToURL("https://proposals.decred.org/record/" + pg.proposal.Token)
```

Comment:
This should not be a static value.. It would be preferable if this connects dynamically to the current Politeia URL either testnet or mainnet URLs.